src/oauth/oauth.controller.ts

```diff
@@ -1,21 +1,30 @@
-import { Body, Controller, Post } from '@nestjs/common';
+import { Body, Controller, Post, Res } from '@nestjs/common';
+import { Response } from 'express';
 import { OauthService } from './oauth.service';
 
 @Controller('oauth')
 export class OauthController {
   constructor(private readonly oauthService: OauthService) {}
 
   @Post('github')
-  async githubLogin(@Body() body): Promise<{ token: string }> {
+  async githubLogin(
+    @Body() body,
+    @Res({ passthrough: true }) response: Response,
+  ): Promise<{ token: string }> {
     const { data } = body;
     const jwtToken = this.oauthService.githubLogin(data.code);
+    response.cookie('jwtToken', jwtToken, { httpOnly: true });
```

Comment:
혹시 cookie에 토큰을 담아보내려고 하시는건가요?? 그렇다면 아래와 같이 수정되어야 할 것 같습니다..!
```suggestion
    response.cookie('jwtToken', (await jwtToken).token, { httpOnly: true });
```